code/Controller/Search.php

```diff
@@ -4,7 +4,13 @@ class Controller_Search extends Controller_Abstract
 {
     public function get()
     {
+
```

Comment:
@bobbyshaw kind of a pet peeve but I'm not really into the leading newline character at the top of the method body :)  I know some people like that.  Maybe we can pick a code style / standard / sniffer or something and just go with that.